### ISSUE TITLE
NTR: Pyridoxal-5'-phosphate low in CSF (Lab: cerebrospinal fluid tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8463]
New term request Pyridoxal-5'-phosphate low in CSF (segal_110420182321): PLP; cerebrospinal fluid; 5-prime

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000206
- label: Reduced CSF pyridoxal-5'-phosphate concentration
- parents: [Abnormal CSF metabolite concentration (HP:0025454)](https://hpo.jax.org/app/browse/term/HP:0025454)
- pmid: PMID:21305354
- def: The concentration of pyridoxal-5'-phosphate in the cerebrospinal fluid (CSF) is below the lower limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8463
- synonyms: Pyridoxal-5'-phosphate low in CSF(NONE)
